golem/task/taskmanager.py

```
@@ -176,6 +176,28 @@ def add_new_task(self, task):
                                  op=TaskOp.CREATED,
                                  persist=False)
 
+    @handle_task_key_error
+    def increase_task_mask(self, task_id: str, num_bits: int = 1) -> None:
+        """ Increase mask for given task i.e. make it more restrictive """
+        task = self.tasks[task_id]
+        try:
+            task.header.mask.increase(num_bits)
+        except AssertionError:
+            return  # Mask cannot be further increased

```

Could you add some logs here? The same goes for `decrease_task_mask`. Now they both fail silently

---

Yes, but that's not really an error. Just reaching max/min size. I thought it would be easier this way but I can as well add some check before increase/decrease and not throw any AssertionError at all.

---

Fixed.